added an accept all option in argparser to answer all questions with default without user input.

osintowl opened the pull request.

Commit:
Added a confirm all option

Added an option to confirm all questions and not prompt for them.

Commit:
Added try catch for input

I know this project is python3 but until python2 is completely out of our lives, atting a try catch here to avoid command injection via input method that python2 was vulnerable to.

orlikoski commented on src/cdqr.py:
```diff
@@ -17,6 +17,11 @@
 except:
     compression = zipfile.ZIP_STORED
 
+try:
```
Can you explain a bit why this was added? It seems it would conflict with those places where the input() function is being used. Such as https://github.com/orlikoski/CDQR/blob/master/src/cdqr.py#L654

osintowl replied:
https://stackoverflow.com/questions/33946678/python-how-to-hack-input

Python 3 does not have a raw_input and it will just except and pass, if anyone tries to run it in python2.7 it will disallow the issue expressed in the stackoverflow above. Until python 2 is officially EOL it is the safer thing to do even for projects like this that are explicitly python3 in case someone accidentally tries to run it in python2.

orlikoski replied:
My understanding is that this works as is in both Py2.x and Py3.x as Py3 has a built in alias from raw_input() to input().

That said, CDQR will break for many other reasons with Py2.x and this project has always been a Python 3 build. Now, a check for python 2 vs 3 and stopping it if it detects Py2.x with an error message instead of this would be a nice addition

osintowl replied:
I can do that! I'll edit this tonight!

osintowl replied:
I just edited it, check the verbiage if you would like. As a side note I used to thing the same thing but there is no raw_input() in python 3, Just verified. That try catch with input/raw_input was a trick I learned from Mark Baggett.  If there are any other additions you have been thinking of adding let me know.

orlikoski replied:
Nice! Slightly changed the wording on the error message and looks ready to go

orlikoski commented:
I like the concept and just have a couple of questions to dig into

Commit:
Update .gitignore

Sorry those were my noisy configs from my ide.

Commit:
Update cdqr.py

Throw an error when not run with python 3. Please check the verbiage and change as needed.

orlikoski committed:
Update cdqr.py

Slightly altered the error message wording

orlikoski reviewed (approved):
Looks good to me
